### PR TITLE
refactor: Migrate PreferencesAccountViewModel to Typescript

### PR DESCRIPTION
**Pull request by thisisamir98**

### CODE REVIEW

File: src/script/view_model/content/PreferencesAccountViewModel.ts [hunk 1]
```diff
@@ -119,14 +147,14 @@ z.viewModel.content.PreferencesAccountViewModel = class PreferencesAccountViewMo
...
-    this.teamName = ko.pureComputed(() => t('preferencesAccountTeam', this.teamRepository.teamName()));
+    this.teamName = ko.pureComputed(() => t('preferencesAccountTeam', this.teamRepository.getTeamName()()));
 
     this.optionPrivacy = ko.observable();
     this.optionPrivacy.subscribe(privacyPreference => {
       this.propertiesRepository.savePreference(PROPERTIES_TYPE.PRIVACY, privacyPreference);
     });
 
-    this.optionReadReceipts = this.propertiesRepository.receiptMode;
+    this.optionReadReceipts = this.propertiesRepository.getReceiptMode();
```
Comment (bennycode) [inline src/script/view_model/content/PreferencesAccountViewModel.ts]: Why using a method call here instead of using the property directly?

Comment (thisisamir98) [reply src/script/view_model/content/PreferencesAccountViewModel.ts]: receiptMode is a private member in PropertiesRepository

Comment (bennycode) [reply src/script/view_model/content/PreferencesAccountViewModel.ts]: You can make it a read-only public member.

Comment (thisisamir98) [reply src/script/view_model/content/PreferencesAccountViewModel.ts]: https://github.com/wireapp/wire-webapp/pull/9073/commits/20aa1d2943e5dcd65dd596d8c9200ea557d1f82d

**Pull request merged**